src/Symfony/Bundle/TwigBundle/CacheWarmer/TemplateCacheCacheWarmer.php

```diff
@@ -27,14 +29,16 @@ class TemplateCacheCacheWarmer implements CacheWarmerInterface
 {
     protected $container;
     protected $finder;
+    protected $paths;
```

Comment:
I typically would make it private but was following the convention of the file. Should I make it private then?